docs/eventing/samples/writing-receive-adapter-source/01-theory.md

```diff
@@ -83,22 +83,21 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
         // ...
 }
 ```
-Ensure that the specific source subdirectory has been added to the injection portion of the `hack/update-codegen.sh` script.
-
-```patch
-
-# Sources
-+API_DIRS_SOURCES=(github/pkg camel/source/pkg kafka/source/pkg awssqs/pkg couchdb/source/pkg prometheus/pkg YourSourceHere/pkg)
--API_DIRS_SOURCES=(github/pkg camel/source/pkg kafka/source/pkg awssqs/pkg couchdb/source/pkg prometheus/pkg)
-
-```
-and
-```patch
-  -i knative.dev/eventing-contrib/camel/source/pkg/apis \
-- -i knative.dev/eventing-contrib/github/pkg/apis
-+ -i knative.dev/eventing-contrib/github/pkg/apis \
-+ -i knative.dev/eventing-contrib/YourSourceHere/pkg/apis
```

Comment:
This part is now explained in a separate page: https://github.com/knative/docs/pull/2475/files#diff-7f8a6b25a6b73452947bc5e926d1e6c9R1